crates/astria-sequencer/src/state_ext.rs

```diff
@@ -167,11 +179,31 @@ pub(crate) trait StateReadExt: StateRead {
 
 impl<T: StateRead> StateReadExt for T {}
 
+fn revision_number_from_chain_id(chain_id: &str) -> u64 {
```

Comment:
If I supply a chain ID `123`, won't this function parse the wrong thing? Alternatively, what about using regex groups to be stricter about how this is parsed?

Can you add unit tests for the cases `123` and `123-123`?

Reply:
removed in favour of regex and added tests